test/parallel/test-http-mutable-headers.js

```diff
@@ -43,8 +43,9 @@ const s = http.createServer(common.mustCall((req, res) => {
   switch (test) {
     case 'headers':
       // Check that header-related functions work before setting any headers
-      // eslint-disable-next-line no-restricted-properties
-      assert.deepEqual(res.getHeaders(), {});
+      const headers = res.getHeaders();
+      const objLike = Object.create(null);
```

Comment:
I had this discussion with @addaleax. How can it be an `Object` if `objLike instaceof Object === false` ?
According to OOP definitions  "not" `objLike` is an `Object`. It's actually a `null` with properies 😉